MonitorControl.xcodeproj/xcshareddata/xcschemes/MonitorControl.xcscheme

```diff
@@ -43,6 +43,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "es-419"
```

Comment:
```suggestion
```
This sets the default scheme language to Spanish, so it should be reverted :) (probably happened during testing)